client/src/routes/(app)/auth/datapass/create-organization/+page.svelte

```diff
@@ -15,7 +16,10 @@
         <p>
           Vous ne pouvez pas accéder au service pour le moment, mais vous pouvez
           effectuer une demande d’enregistrement afin d’intégrer votre
-          organisation à notre outil.
+          organisation à notre outil. Pour obtenir plus de détails, veuillez
+          consulter <a target="_blank" href={USER_DOCUMENTATION_LINK}
+            >la documentation détaillée</a
+          >.
```

Comment:
Il faut utiliser `class="fr-link"` pour que le lien s'affiche en bleu : https://gouvfr.atlassian.net/wiki/spaces/DB/pages/217284725/Liens+-+Links#Lien-au-fil-du-texte

Par ailleurs :

> À noter : l'attribut target="_blank"  permettant l’ouverture d’une nouvelle fenêtre doit être accompagné d'un attribut rel="noopener" (et "noreferrer" sur les vieux navigateurs), en particulier si le lien pointe sur un site externe, afin d'éviter toute possibilité d’exécution de code malveillant (l’article de référence (en anglais) :  [https://mathiasbynens.github.io/rel-noopener/)](https://mathiasbynens.github.io/rel-noopener/)

https://gouvfr.atlassian.net/wiki/spaces/DB/pages/217284725/Liens+-+Links#Lien-au-fil-du-texte

Donc :

```suggestion
          consulter <a class="fr-link" target="_blank" rel="noopener" href={USER_DOCUMENTATION_LINK}
            >la documentation détaillée</a
          >.
```